### DISCUSSION TITLE
Update: final evaluations and NeurIPS event

### DISCUSSION THREAD

**Discussion by gasse** [ds4dm/ml4co-competition#106] [Announcements]
Dear participants,

Here are the latest news regarding the competition.

### Final evaluations

It's been a couple of weeks since we've received your final submissions, and so far things have been running smoothly for most of the teams. Only a few submissions have resulted in OOM errors (in the primal and config tasks), and a few others have resulted in a failure during execution. We'll get back to the concerned teams shortly to try and find a workaround when possible.

Below is a list of the teams competing in each of our challenges. If your team name does not appear in this list and you've made a submission, then please get back to us.

**Primal task (13 submissions)**

- CUHKSZ_ATD
- dt
- EasyML
- EI-OROAS
- ethz-scuba
- generaleyes
- MDO
- MILPandCookies
- MixedInspiringLamePuns
- Mr_Tree
- nf-lzg
- Nuri
- UNIST-LIM-Lab

**Dual task (24 submissions)**

- ark
- blueterrier
- Columbia_Combinators
- comeon
- CUHKSZ_ATD
- DaShun
- EasyML
- EFPP
- EI-OROAS
- gentlemenML4CO
- KAIST_OSI
- KEP-UNIST
- lxj24
- Monkey
- nf-lzg
- null_
- Nuri
- qqy
- shg
- Superfly
- sylvia
- THUML-RL
- uofx
- wanghaha

**Config task (15 submissions)**

- Cho
- DaShun
- EasyML
- ECNUNoah001
- EI-OROAS
- KAIST_OSI
- KEP-UNIST
- MDO
- MixedInspiringLamePuns
- MOEA
- nf-lzg
- null_
- Nuri
- shg
- THUML-RL

### NeurIPS event

There has been a change of schedule to the NeurIPS event. The competition's live presentation is now scheduled to Thursday, December 9th (instead of Friday, December 10th), at ~~18:25 GMT~~ 18:45 GMT. We will have a 20 minutes live presentation in the main competition track, immediately followed by a 3 hours Zoom + Gathertown breakout session dedicated to the competition, which will take the form of a workshop-like event. The program is the following

- ~~18:25~~ 18:45 - 20 min live presentation in the NeurIPS competition track
- ~~18:45~~ 19:05 - 3h virtual event on Zoom / Gathertown
    - ~~18:45~~ 19:05 - short wrap-up (15 min)
    - ~~19:00~~ 19:20 - short presentation from the winning teams (3x10 min)
    - ~~19:30~~ 19:50 - poster session on Gathertown (30 min)
    - ~~20:00~~ 20:20 - break (15 min)
    - ~~20:15~~ 20:35 - keynote (30 min) - Kate Smith-Miles
    - ~~20:45~~ 21:05 - panel discussion (1 hour max) - Andrea Lodi, Timo Berthold, Frank Hutter, Vinod Nair, Kevin Leyton-Brown

We've been asked by some of you whether a NeurIPS registration is needed to attend the virtual event, and unfortunately the answer is yes. We've asked the question to the NeurIPS organizers, and have been answered the following:

> Regarding NeurIPS registrations and participation to the breakout sessions: all people who participate (both speakers and audience) need to have a paid registration, and hence to sign the NeurIPS code of conduct. Zoom links to the sessions should not be shared to anyone, and should only be accessed from the conference portal after paying the registration and agreeing to the code of conduct.

If you have any question or comment at this point, please feel free to reply to this discussion.

Thanks again for your interest and participation. We wish to see you soon !

Best,
The organizing team

**Comment by superboom** [comment]
The result announcement will be delay?
